Makefile

```diff
@@ -206,6 +206,7 @@ util/build_version.cc: FORCE
 	else mv -f $@-t $@; fi
 
 LIBOBJECTS = $(LIB_SOURCES:.cc=.o)
+LIBOBJECTS += $(TOOL_SOURCES:.cc=.o)
```

Comment:
Tool sources were already included in the library, so this is not adding any new bloat. (See src.mk, where TOOL_SOURCES is a new variable and the contents used to be a part of LIB_SOURCES.)

They could be removed, I think, but lots of rules in the Makefile assume that they're there, so the changes became significant when I tried to separate them completely.